docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -0,0 +1,374 @@
+| `manage-users`           | A realm administrator can `view`, `map-roles`, `manage-group-membership` and `manage` all users in the realm, 
+                             as well as `view`, `manage-membership` and `manage` groups in the realm.
+| `impersonation`          | A realm administrator can `impersonate` all users in the realm.
+| `view-clients`           | A realm administrator can `view` all clients in the realm.
+| `manage-clients`         | A realm administrator can `view` and `manage` all clients in the realm. As well as
+                             `map-roles-client-scope`.
+                             TODO: review the expected behavior: `RolePermissions.checkAdminRoles(RoleModel)`
+                             TODO: `map-roles`, `map-roles-composite`.
+|===
+
+==== Understanding some common use cases
+
+Let's say an admin wants to allow a group of admins to `manage` all users in the realm except those, which belongs to the
+administrators group. In our example, we have a realm called `test` and a group called `test-admins`. 
+
+===== Allow manage users by group of admins
+
+Create user permission permission, allowing to view and manage all users in the realm for members of the `test-admins` group:
+
+* Navigate to the `Permissions` tab in the Admin Console.
+* Click `Create permission` button and choose `Users` resource type
+* Fill the name, e.g. `"Allow managing all realm users"`
+* Choose `view` and `manage` authorization scopes, keep checked `All Users`
+* Create a condition, which needs to be met to get an access by clicking `Create new policy`
+* Fill name `"Allow test-admins"`, select `Group` as `Policy type`. 
+* Click `Add groups` button and select `test-admins` group, click `Save`
+* Click `Save` on `"Create permission"` page
+
+===== Allow manage users by group of admins but not group members
+
+Let's exlude the members of the group itself, so that `test-admins` cannot manage other admins.
+
+* Create new permission by clicking `Create permission` button
+* This time choose `Groups` resource type
+* Fill the name, e.g. `"Disallow managing test-admins"`
+* Choose `manage-members` authorization scope.
+* Select `Specific Groups` and choose `test-admins` group
+* `Create new policy` of type `Group`
+* Fill the name `"Disallow test-admins"` and select `test-admins` group
+* Switch to `Negative Logic` for the policy, `Save` the policy
+* `Save` the permission
+
+=====  Allow impersonation of users for members of admin group with specific role assigned
+
+- Create a **User Permission** for specific users (or all users) you want to allow impersonation.
+- Create a **Group Policy** allowing access to members of `test-admins`.
+- Create a **Role Policy** allowing access to users assigned the `impersonation-admin` role.
+- Assign both policies to the permission.
+
+===== Blacklist specific users from being impersonated
+
+- Create a **User Permission** for the specific users you want to prevent from being impersonated.
```

Comment:
```suggestion
- Create a *User Permission* for the specific users you want to prevent from being impersonated.
```

Reply:
You only need two asterisks in very specific situation, such as when the asterisk is followed by a colon, semi-colon, or special formatting characters.